_posts/2018-04-01-from-chemical-engineering-to-software-development.md

```diff
@@ -0,0 +1,114 @@
+# From chemical engineering to software development
+
+1st of April, 2016, Paris, _Gare du Nord_. With my backpack and my monitor in a bag, I onboard a Eurostar train to London with my cousin Tom and his wife Emma. I leave a chemical engineer position behind me, for a software developer one.
+
+The year is now 2018, I work for a company called [Driftrock](https://www.driftrock.com/): we boost the marketing power of our clients through social media. I'm a software developer there, taking care of an awesome microservices fleet with an heavily-skilled team and juggling between back-end, front-end and DevOps, using various programming languages to achieve our goals. Before that, I have spent more than a year at [The App Business](http://www.theappbusiness.com/), a mobile applications agency, where I was a junior software engineer, learning key concepts like Agile, testing and clean code.
+
+Time for a check-up after 2 years, isn't it?
+
+I want to share here some insights on my career switch. Personal opinions and experiences are exposed, in the most honest way possible, far be it from me to bash the chemical engineering industry, nor force any chemical engineers to shift away from their jobs! I am still keen to take any feedback and comments if you disagree.
+
+Unfortunately, I give here only the differences between a non-tech industry and the software industry, therefore, the purpose of this article is not about how to start your software engineer career (however, you can still drop me a message if you are looking for any pieces of advice).
+
+The differences between the British and the French way of working are not tackled here, it is a wide subject deserving a full and proper article (trailer: I am writing from London, which gives you a good idea of my humble opinion).
+
+## Where it all began...
+
+With few words, the `Once upon a time` is: I grew up on a French island ([Reunion Island](https://www.google.co.uk/maps/place/Reunion/@-21.1351121,55.2471124,10z/data=!3m1!4b1!4m5!3m4!1s0x2178778110b8e43b:0x4a7f8e89ecdbeaf9!8m2!3d-21.115141!4d55.536384) - oversea territory), did a normal scholarship until my _baccalauréat_ (UK GCSE equivalent), attended to intensive courses in mathematics, physics and chemistry for 2 years (_Classe Preparatoire aux Grandes Ecoles_), was admitted into a national chemical engineering school ([ENSIC](https://ensic.univ-lorraine.fr/)) in the lovely town of Nancy (the Lorraine region, in the mainland), graduated after 3 years and found my first job as a chemical engineer in Paris.
+
+From that point, the career path was almost sealed: I would have spent the next decades in the same company (because it shows loyalty and mind diversity is not really good you know?), painfully risen through the ranks, fought against this silly bureaucracy where more senior people teach you what to do in a world they can no longer understand, gone in retirement at 70 years old with nothing (stay lucid, this is going to happen).
```

Comment:
- 'mind diversity is not really good you know' - do you have an example of this? it comes across as quite subjective. From my experience, I don't think the problem is a discouragement of broadening your mind, but often it's less of a necessity in chem eng, given that you're often dealing with well-established techniques and processes, and usually there's a tried and tested way of getting things done in order to meet whatever health and safety, technical, quality requirements, and hence there's not much need for divergence...
- 'where more senior people teach you what to do in a world they can no longer understand' - woah! bit harsh :P perhaps something more along the lines of: 'where seniority often results in adopting a more managerial role and becoming removed from technical processes' and perhaps 'and hence become disconnected from technological advances.' However you want to write it, I think this should be toned-down or elaborated on to explain why you think like this!